IdentityCore/src/configuration/webview/MSIDWebviewConfiguration.h

```diff
@@ -27,6 +27,10 @@
 
 #import <Foundation/Foundation.h>
 
+#if TARGET_OS_IPHONE
+#import <UIKit/UIKit.h>
+#endif
+
```

Comment:
Why do we need this? :)

Reply:
because of these 2 properties:

```
#if TARGET_OS_IPHONE
@property (weak) UIViewController *parentController;
@property (readwrite) UIModalPresentationStyle presentationType;
#endif
```

We need to include UIModalPresentationStyle & UIViewController from uikit.